docs/_data/atomics.json

```diff
@@ -1334,5 +1334,37 @@
       "define": "<p class='mb0'>Remove a transition delay from an element.</p>",
       "responsive": false
     }
+  ],
+  "cropping": [
+    {
+      "class": ".of-contain",
+      "output": "object-fit: contain",
+      "define": "<p class='mb0'>Resize the image, while maintaining its aspect ratio, so that the whole image fits inside the container. This may result in some empty space in the container.</p>"
+    },
+    {
+      "class": ".of-cover",
+      "output": "object-fit: cover",
+      "define": "<p class='mb0'>Resize the image, while maintaining its aspect ratio, so that the whole container is filled. The image may get cropped as a result.</p>"
+    },
+    {
+      "class": ".of-fill",
+      "output": "object-fit: fill",
+      "define": "<p class='mb0'>Stretch or squash the image, so that the whole container is filled. The image's aspect ratio may change as a result.</p>"
+    },
+    {
+      "class": ".of-none",
+      "output": "object-fit: none",
+      "define": "<p class='mb0'>Don't resize the image at all. This may result in the image getting cropped, or there being some empty space in the container.</p>"
+    },
+    {
+      "class": ".of-scale-down",
+      "output": "object-fit: scale-down",
+      "define": "<p class='mb0'>If the image fits inside the container, it's not resized. If it's too large, it will be scaled down just like <code>.of-contain</code>.</p>"
+    },
+    {
+      "class": ".op-center",
+      "output": "object-position: 50% 50%",
+      "define": "<p class='mb0'>Positions the image in the centre of the container.</p>"
```

Comment:
> centre

@benjamin-hodgson are you British?

Reply:
Yes

Reply:
Sadly yes. :trollface: 

But! We have [opinions](https://stackoverflow.design/content/guidelines/grammar-and-mechanics/#use-american-spelling-and-standards) on this. 🇺🇸

Reply:
```suggestion
      "define": "<p class='mb0'>Positions the image in the center of the container.</p>"
```